src/state/apis/snapshot/snapshot.ts

```diff
@@ -94,6 +106,36 @@ export const snapshotApi = createApi({
         }
       },
     }),
+    getThorStrategies: build.query<Strategy[], void>({
+      keepUnusedDataFor: Number.MAX_SAFE_INTEGER, // never refetch these
+      queryFn: async (_, { dispatch }) => {
+        const query = `
+          query {
+            space(id: "${THORSWAP_SNAPSHOT_SPACE}") {
```

Comment:
We could pass this as a parameter and change the logic, but pretty nitpick from my pov as we will want to kick this out in a few month